src/server/SocketServer.java

```diff
@@ -14,13 +21,22 @@
  * continuous availability for client connections until shutdown.
  */
 public class SocketServer {
+    /**
```

Comment:
Should have a blank line between the start of the class and the first field

```suggestion
public class SocketServer {

    /**
```